Octokit/Models/Response/Team.cs

```diff
@@ -71,9 +72,14 @@ public Team(string url, string htmlUrl, int id, string nodeId, string slug, stri
         public StringEnum<TeamPrivacy> Privacy { get; private set; }
 
         /// <summary>
-        /// permission attached to this team
+        /// Deprecated. The permission that new repositories will be added to the team with when none is specified
```

Comment:
I'm not sure this is right. This model is returned from `gitHubClient.Repository.GetAllTeams(repositoryOwner, repositoryName)`, so the permission is actually "what permission does this team have for this repo".

Reply:
(Previously `PermissionLevel` was wrong too)

Reply:
@matt-richardson you are correct! This comment is more suitable for NewTeam.cs and UpdateTeam.cs than the Team.cs.

From the API reference [here](https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#get-a-team-by-name), the permission has a description of "Permission that the team will have for its repositories", which I think would be a better comment for this property. I can submit a quick patch updating the comment if we can agree on that.

I am currently working on a [PR] (https://github.com/octokit/octokit.net/pull/2661/files#diff-2485e4802b12efd4be9bf3a359200af95b4c8ce7306db1ea5dfa9f44b8e225b0) that takes care of all the other outdated permissions.